CHANGELOG.md

```diff
@@ -2,6 +2,18 @@
 
 Records breaking changes from major version bumps
 
+## 6.0.0
+
+PR: [#31](https://github.com/alphagov/digitalmarketplace-apiclient/pull/31)
+
+### What changed
+
+Removed `update_brief_status` method.
+
+This functionality has been removed and calls to publish a brief (i.e. change it's status from
+'draft' to 'live') should now use the `publish_brief` method. There should no other current
+uses of the `update_brief_status` in our apps that set a status to something other than 'live'.
+
```

Comment:
` ### Example app change` seems to have been lost in the past few updates, but I think we can do one for this to show how the method name and arguments should be changed.